### PR TITLE
[Fix] Stories with API requests

### LINKED ISSUE #9620: 🐛 API stories broken
## 🐛 Bug

After the migration to `client-preset` our operation names slightly changed. Some stories were missed in changing their mock requests to match the operation name changes.

## 🦋 Expected Behaviour

All stories that make API requests load as expected

## 🕵️ Details

Stories affected:

- Users table
- IAP Home
- Team members page
- Update user page

## 📋 Steps to Reproduce

1. Run storybook `npm run storybook:web`
2. Navigate to any story listed in the details
3. Observe the story is broken in some way

## 🙋‍♀️ Proposed Solution

Fix the `apiResponses` parameter for those stories to allow the story to load properly.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] Affected stories (see details) are rendering with no errors

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #9620 

## 👋 Introduction

Fixes stories with incorrect API operation names.

## 🕵️ Details

This also fixes an issue where animations were not being loaded after splitting those off from main bundle.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Run storybook `npm run storybook:web`
2. Navigate to the affected stories and confirm they load:
    - User table
    - IAP Home
    - Team members page
    - Update user page

### CODE REVIEW

File: apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx
Comment (mnigh) [inline apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx:1]: why do we have two story files `UpdateUserPage` and `UpdateUserPageNew`?

Comment (mnigh) [inline apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx:1]: this story seems to be missing data in the two Roles and permissions tables.

**Review by mnigh — CHANGES_REQUESTED** [review]

File: apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx
Comment (mnigh) [reply apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx:1]: I would recommend:

1. Delete `/apps/web/src/pages/Users/UpdateUserPage/UpdateUserPage.stories.tsx`
2. Rename `/apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx` to `/apps/web/src/pages/Users/UpdateUserPage/UpdateUserPage.stories.tsx`

Comment (esizer) [reply apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx:1]: Added in [fix update user roles table story data](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9622/commits/0489b440eca40d54bbecf0a0ef3b0980c68adbdd)

Comment (esizer) [reply apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx:1]: Removed old one in [remove redundant story](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9622/commits/f868bc5ac956f2366e51545aa395a67241ba0d3d)

**Review by mnigh — COMMENTED** [review]
🍞 is great. One question about Update User Page stories: https://github.com/GCTC-NTGC/gc-digital-talent/pull/9622#discussion_r1511530205

File: apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx
Comment (esizer) [reply apps/web/src/pages/Users/UpdateUserPage/UpdateUserPageNew.stories.tsx:1]: Looks like we were nulling out a single field and I accidently copy and pasted over it. Not sure if that was super useful so I removed it. We can add it back if we need to 🤷‍♀️ [remove redundant story](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9622/commits/f9a2bc06a8465aa8251c24b12196922c12cd0f1d)

**Review by mnigh — APPROVED** [review]

**Pull request merged**